modules/SetupWizard/Android.mk

```diff
@@ -9,5 +9,6 @@ else
 endif
 
 LOCAL_PRIVILEGED_MODULE := true
+LOCAL_OVERRIDES_PACKAGES := Provision
```

Comment:
ok thanks.
I earlier had disabled its removal on Marshmallow, as it was suggested as a workaround for weird CMSetupwizard behaviour